actions/command.test.js

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import configureStore from 'redux-mock-store';
+
+import thunk from 'redux-thunk';
+
+import * as UserAgent from 'utils/user_agent.jsx';
+import * as GlobalActions from 'actions/global_actions.jsx';
+
+import {executeCommand} from './command';
+const mockStore = configureStore([thunk]);
+
+const currentChannelId = '123';
+const currentTeamId = '321';
+const currentUserId = 'user123';
+const initialState = {
+    entities: {
+        general: {
+            config: {
+                ExperimentalViewArchivedChannels: 'false',
+            },
+        },
+        channels: {
+            currentChannelId,
+        },
+        teams: {
+            currentTeamId,
+        },
+        users: {
+            currentUserId,
+            profiles: {
+                user123: {
+                    timezone: {
+                        useAutomaticTimezone: true,
+                        automaticTimezone: '',
+                        manualTimezone: '',
+                    },
+                },
+            },
+        },
+    },
+    views: {
+        rhs: {
+            rhsState: null,
+            searchTerms: '',
+        },
+    },
+};
+
+jest.mock('utils/user_agent.jsx');
+jest.mock('actions/global_actions.jsx');
+
+describe('executeCommand', () => {
+    let store;
+    beforeEach(async () => {
+        store = await mockStore(initialState);
+    });
+
+    test('should fire the UPDATE_RHS_SEARCH_TERMS with the terms', async () => {
```

Comment:
```suggestion
    test('search should fire the UPDATE_RHS_SEARCH_TERMS with the terms', async () => {
```

Reply:
Alternatively, you could wrap the whole thing in a `describe` block just in case we add more tests to this in the future